content/en/people/index.adoc

```diff
@@ -0,0 +1,27 @@
+---
+title: "People"
+description: "ACL Special Interest Group on Turkic Languages People"
+date: 2020-08-27T19:25:12+02:00
+lastmod: 2020-08-27T19:25:12+02:00
+draft: false
+images: []
+contributors: []
+---
+
+President:: Duygu Ataman mailto:sigturk-president@gmail.com[(email)]
+
+Secretary:: Sardana Ivanova mailto:sigturk-secretary@gmail.com[(email)]
+
+
+== Advisory Committee
+
+
+== Organization Contributors
+
+* Mehmet Oguz Derin link:++https://github.com/mehmetoguzderin++[(GitHub)]
+* Mammad Hajili link:++https://github.com/mammadhajili++[(GitHub)]
+* Jamshidbek Mirzakhalov link:++https://github.com/mirzakhalov++[(GitHub)]
+* Jonne Sälevä link:++https://github.com/j0ma++[(GitHub)]
+
+== Code Contributors
```

Comment:
Is this line necessary given that it's empty? If there is content to be added, can we add it?

Reply:
We do not have non-org non-officer contributors yet (similar story for Advisory Committee), but request on issue was to prep these sections